ownership-chain/precompile/evolution-collection/contracts/EvolutionCollection.sol

```diff
@@ -57,4 +63,17 @@ interface EvolutionCollection {
         uint256 _tokenId,
         string calldata _tokenURI
     ) external;
+
+    /// @notice Enables public minting for the collection
```

Comment:
explain below: 
// When enabled, any address is allowed to mint on this collection
// This does not affect evolution: only the owner of the collection can continue evolving assets